iwyu.cc

```diff
@@ -3483,17 +3490,17 @@ class IwyuAstConsumer
       AddUsingDeclaration((*it)->getTargetDecl(), decl);
```

Comment:
Maybe rename `AddUsingDeclaration` to `AddUsingShadowDecl`? Because now we have `AddUsingDeclaration` and `AddUsingDecl` which is confusing.